api/v1/helpers/verbs/findUtils.js

```diff
@@ -145,9 +145,12 @@ function toSequelizeOrder(sortOrder, modelName) {
 function options(params, props) {
   const opts = u.buildFieldList(params);
 
-  // Specify the sort order
-  const ord = (params.sort ? params.sort.value : null) || props.defaultOrder;
-  opts.order = toSequelizeOrder(ord, props.modelName);
+  // Specify the sort order. If defaultOrder is defined in props then
+  // update sort order otherwise take value from model defination
+  if (props.defaultOrder) {
+    const ord = (params.sort ? params.sort.value : null) || props.defaultOrder;
```

Comment:
No... it should still sort by params.sort.value even if props.defaultOrder is empty or undefined.